api/CanvasRenderingContext2D.json

```diff
@@ -1117,116 +1117,6 @@
           }
         }
       },
-      "currentTransform": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/currentTransform",
-          "support": {
-            "chrome": {
-              "version_added": true,
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "Experimental Web Platform Features"
-                }
-              ]
-            },
-            "chrome_android": {
-              "version_added": false
-            },
-            "edge": {
-              "version_added": "≤18"
```

Comment:
Was it in Edge? I'm not sure what we should do with things that were in IE or EdgeHTML but not in Chromium, Gecko or WebKit. That's not really a well explored part of the removal guideline, right?

Reply:
Well, the removal guidelines say no browser versions released in the past two years, so since Edge 18 was released October 2018, I'd say we're good?

Reply:
If we apply this guideline fully, won't we be removing a bunch of APIs that IE/Edge have which are necessary alternatives to other APIs which yet aren't simple aliases of the cross-browser APIs?

Listing all APIs which could be removed under the guideline might help show if there's an issue like this.

Reply:
I'd prefer to keep this data until at least 15 Jan 2022, another year.

The relevant bit of the irrelevance feature guideline says a feature can be removed if it "was implemented and has since been removed from all browsers dating back two or more years ago." I admit the wording is a bit awkward, but it's supposed to mean that two years have elapsed since the removal, not the ship date of the last version containing that feature (otherwise we wouldn't have bothered with the subsequent bullet in the guideline, with the five year horizon).